### ISSUE TITLE
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/Reduce

### ISSUE THREAD

**Issue by GFore** [mdn/sprints#849]
# Request type
- [ ] New documentation
- [ X] Correction or update

## Page to change
If this request relates to an existing article, please give its URL:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/Reduce#Running_Promises_in_Sequence

# Details
*Please describe the change you are requesting. If you know of technical experts who can review the change, please mention them.*
The last line of the example code in the Running Promises in Sequence section seems to have an error since no argument is provided for console.log. When I run the code it does not output anything to the console. I think that this line:
`.then(console.log);`
should be changed to something like this instead:
`.then(x => console.log(x));`

**Comment by duduindo** [comment]
@GFore  Which version of your web browser?

**Comment by GFore** [comment]
I'm just using the standard self-updating Chrome browser, so whichever version was in place on Jan 23, 2019, I suppose. Are there any browser scenarios where 'console.log' will provide output to the console when no argument is supplied to it? I'm no expert on this - I just know that I saw no output until I made the change that I suggested above.

**Comment by GFore** [comment]
I stand corrected - I tried running the code again today as shown in the doc page I linked to above (using Chrome71) and it does correctly output 1200.  Not sure if there was a recent change in Chrome that makes this work or if I did something wrong when I tried to run the code a few weeks ago. In any case, this if the first time I've seen console.log used this way (implicitly taking as an argument what was returned from a promise). I learn something new about JS every day. :)
